app/assets/javascripts/student_profile_v2/profile_details.js

```diff
@@ -22,55 +54,113 @@
 
       _.each(this.props.attendanceData.tardies, function(obj){
         events.push({
-          message: name + ' was tardy.', date: Date.parse(obj.occurred_at)
+          type: 'Absence',
+          compressed: true,
+          message: name + ' was tardy.',
+          date: Date.parse(obj.occurred_at),
+          color: Colors.procedural
         });
       });
       _.each(this.props.attendanceData.absences, function(obj){
         events.push({
-          message: name + ' was absent.', date: Date.parse(obj.occurred_at)
+          type: 'Tardy',
+          compressed: true,
+          message: name + ' was absent.',
+          date: Date.parse(obj.occurred_at),
+          color: Colors.procedural
+        });
+      });
+      _.each(this.props.attendanceData.discipline_incidents, function(obj){
+        events.push({
+          type: 'Incident',
+          message: obj.incident_description + ' in the ' + obj.incident_location,
+          date: Date.parse(obj.occurred_at),
+          color: Colors.procedural
         });
       });
       _.each(this.props.chartData.mcas_series_ela_scaled, function(quad){
         var score = quad[3];
         events.push({
+          type: 'MCAS ELA',
           message: name + ' scored a ' + score + ' on the ELA section of the MCAS.',
+          color: Colors.reading,
           date: QuadConverter.toDate(quad)
         });
       });
       _.each(this.props.chartData.mcas_series_math_scaled, function(quad){
         var score = quad[3];
         events.push({
+          type: 'MCAS Math',
           message: name + ' scored a ' + score + ' on the Math section of the MCAS.',
+          color: Colors.math,
           date: QuadConverter.toDate(quad)
         });
       });
       _.each(this.props.chartData.star_series_reading_percentile, function(quad){
         var score = quad[3];
         events.push({
-          message: name + ' scored a ' + score + '% on the Reading section of STAR.',
-          date: QuadConverter.toDate(quad)
+          type: 'STAR Reading',
+          message: name + ' scored in the ' + score + 'th percentile on the Reading section of STAR.',
+          date: QuadConverter.toDate(quad),
+          color: Colors.reading
         });
       });
       _.each(this.props.chartData.star_series_math_percentile, function(quad){
         var score = quad[3];
         events.push({
-          message: name + ' scored a ' + score + '% on the Math section of STAR.',
-          date: QuadConverter.toDate(quad)
+          type: 'STAR Math',
+          message: name + ' scored in the ' + score + 'th percentile on the Math section of STAR.',
+          date: QuadConverter.toDate(quad),
+          color: Colors.math
+        });
+      });
+      _.each(this.props.feed.deprecated.interventions, function(obj){
+        events.push({
+          type: 'Note',
+          message: _.template("<%=name%> (Goal: <%=goal%>)")(obj),
```

Comment:
Let's stick with using React for rendering HTML rather than Underscore's templating.  This looks like just string concatenation, so you can do:

```
message: obj.name + '(Goal: ' + obj.goal + ')'
```